templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -12,6 +12,21 @@ Parameters:
       - 'PostgreSQL'
     ConstraintDescription: Must be 'Amazon Aurora PostgreSQL' or 'PostgreSQL'.
     Type: String
+  DBAuroraPostgresEngineVersion:
+    Default: 9.6.12
+    AllowedValues:
```

Comment:
Do you reckon we could use one parameter for postgres version and then use a mapping to pass the correct latest patch version for the given version to each nested template?

Reply:
I think that is definitely going to be needed, especially considering we have exceeded the param limit of 60 for the jira templates when these two parameters are added. Infact jira is sitting at 60 params as it stands, so there may need to be some thinking around how we can facilitate this